README.md

```diff
@@ -26,220 +35,196 @@ required for testing citus.
 
-## <a name="basic-cluster-setup"></a> Basic Cluster Setup
+You should have `az cli` in your local to continue.
 
-On your local machine:
-```bash
+You should use `ssh-agent` to add your ssh keys, which will be used for downloading the enterprise repository. Note that your keys are kept only in memory, therefore this is a secure step.
 
-# Add your EC2 keypair's private key to your agent
-ssh-add path_to_keypair/metin-keypair.pem
+In `azuredeploy.parameters.json` file, you will see the parameters that you can change. For example if you want to change the number of workers, you will need to change the parameter `numberOfWorkers`. You can change the type of coordinator and workers separately from the parameters file. Also by default for workers, memory intense vms are used(E type) while for coordinator CPU intense vms are used(D type).
 
-# Quickly start a cluster of (1 + 2) c3.xlarge nodes at availability zone us-east-1b
-cloudformation/create-stack.sh -k metin-keypair -s FormationMetin -n 2 -i c3.xlarge -a us-east-1b
+After you run tests, you can see the results in `results` folder. The `results` folder will have the name of the config used for the test.
 
-# When your cluster is ready, it will prompt you with the connection string, connect to master node
-ssh -A ec2-user@ec2-35-153-66-69.compute-1.amazonaws.com
-```
+The data will be stored on the attached disk, size of which can be configured in the parameters.
 
-On the coordinator node:
-```bash
-# Setup your test cluster with PostgreSQL 10.1 and Citus master branch
-fab use.postgres:10.2 use.citus:master setup.basic_testing
+If you dont specify the region, a random region among `eastus`, `west us 2` and `south central us` will be chosen. This is to use resources uniformaly from different regions.
 
-# Lets change some conf values 
-fab pg.set_config:max_wal_size,"'5GB'"
-fab pg.set_config:max_connections,1000
+## <a name="azure-setup-steps"></a> Setup Steps For Each Test
 
-# And restart the cluster
-fab pg.restart
-```
+You will need to follow these steps to create a cluster and connect to it, on your local machine:
 
-On your local machine:
 ```bash
-# Delete the formation
-# It's a good practice to check deletion status from the cloud formation console
-aws cloudformation delete-stack --stack-name "FormationMetin"
 
-```
+# start ssh agent
+eval `ssh-agent -s`
 
-## <a name="pgbench"></a> Running PgBench Tests
+# Add your Github ssh key for enterprise (private) repo
+ssh-add
 
-On your local machine:
-```bash
+# in the session that you will use to ssh, set the resource group name
+export RESOURCE_GROUP_NAME=give_your_name_citus_test_automation_r_g
 
-# Add your EC2 keypair's private key to your agent
-ssh-add path_to_keypair/metin-keypair.pem
+# if you want to configure the region
+export REGION=eastus2
```

Comment:
super Minor: I think we should  keep but comment out `export REGION=eastus2`, otherwise everyone would follow this and we wouldn't be able to distribute the load across regions.

Reply:
makes sense